### PR TITLE
PARSEC Page added to DevHub

### PR DESCRIPTION
**Pull request by david-maidsafe**
Added a new 'Stand-Alone Technology' section to the DevHub front page and a new page to host PARSEC information.

### CODE REVIEW

File: src/contents/en-gb/home.yaml [hunk 1]
```diff
@@ -46,6 +46,18 @@ contribute:
     link:
       name: Core Technology
       href: /core_technology
+sharing:
+  title:
+    shared: Stand alone
```
Comment (nadiaburborough) [inline src/contents/en-gb/home.yaml]: The alignment of 'Stand alone technology' title is too low (at full width screen on laptop

File: src/contents/en-gb/home.yaml [hunk 2]
```diff
@@ -46,6 +46,18 @@ contribute:
...
+    tech: technology
+    safe: SAFE Network
+    desc: Use standalone components from the SAFE Network in your own projects.
+  parsec:
+    title: Protocol for Asynchronous, Reliable, Secure and Efficient Consensus
+    desc: PARSEC is an algorithm which allows a section of network nodes to reach consensus on the validity and order of network events by voting on events and using a gossip protocol to disseminate these votes amongst themselves.
```
Comment (nadiaburborough) [inline src/contents/en-gb/home.yaml]: This description goes right to the right hand edge of the screen on a laptop screen at full size.

File: src/contents/en-gb/parsec.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+pageIntro:
+  title: What is PARSEC?
+  desc: PARSEC is a Protocol for Asynchronous, Reliable, Secure and Efficient Consensus and is the world's first open source ABFT protocol that can be used in a truly permissionless setting created by MaidSafe.
```
Comment (nadiaburborough) [inline src/contents/en-gb/parsec.yaml]: font here doesn't match the rest of the page

File: src/contents/en-gb/parsec.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
+pageIntro:
+  title: What is PARSEC?
+  desc: PARSEC is a Protocol for Asynchronous, Reliable, Secure and Efficient Consensus and is the world's first open source ABFT protocol that can be used in a truly permissionless setting created by MaidSafe.
+networkUse:
+  title: Why does the Network need it?
+  para1: As a global decentralised data and communications network, the SAFE Network has to be public and permissionless.
+  para2: Unlike centralised systems there is no central body to act as a final authority on the state of the Network.  Nodes is a decentralised system such as the SAFE Network must therefore agree on Network actions and state amongst themselves. Other decentralised systems such as BitCoin use blockchain to achieve consensus.  However, there is a number of drawbacks with a blockchain based system notably the speed and cost as blockchain grows - this is not a scalable option.
```
Comment (nadiaburborough) [inline src/contents/en-gb/parsec.yaml]: Needs a space above this para

Comment (nadiaburborough) [reply src/contents/en-gb/parsec.yaml]: Nodes ~is~ `on` a decentralised

Comment (nadiaburborough) [reply src/contents/en-gb/parsec.yaml]: therefore agree on ~N~ `n`etwork actions

Comment (nadiaburborough) [reply src/contents/en-gb/parsec.yaml]: However, there ~is~ `are` a

Comment (nadiaburborough) [reply src/contents/en-gb/parsec.yaml]: cost as ~blockchain~ `it` grows

File: src/contents/en-gb/parsec.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
+pageIntro:
+  title: What is PARSEC?
+  desc: PARSEC is a Protocol for Asynchronous, Reliable, Secure and Efficient Consensus and is the world's first open source ABFT protocol that can be used in a truly permissionless setting created by MaidSafe.
+networkUse:
+  title: Why does the Network need it?
+  para1: As a global decentralised data and communications network, the SAFE Network has to be public and permissionless.
+  para2: Unlike centralised systems there is no central body to act as a final authority on the state of the Network.  Nodes is a decentralised system such as the SAFE Network must therefore agree on Network actions and state amongst themselves. Other decentralised systems such as BitCoin use blockchain to achieve consensus.  However, there is a number of drawbacks with a blockchain based system notably the speed and cost as blockchain grows - this is not a scalable option.
+bigDeal:
+  title: Why is it a big deal?
```
Comment (nadiaburborough) [inline src/contents/en-gb/parsec.yaml]: Why is ~it~ `PARSEC` a big deal?

File: src/contents/en-gb/parsec.yaml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+networkUse:
+  title: Why does the Network need it?
+  para1: As a global decentralised data and communications network, the SAFE Network has to be public and permissionless.
+  para2: Unlike centralised systems there is no central body to act as a final authority on the state of the Network.  Nodes is a decentralised system such as the SAFE Network must therefore agree on Network actions and state amongst themselves. Other decentralised systems such as BitCoin use blockchain to achieve consensus.  However, there is a number of drawbacks with a blockchain based system notably the speed and cost as blockchain grows - this is not a scalable option.
+bigDeal:
+  title: Why is it a big deal?
+  para1: As far as we know no other project has managed to ensure ABFT consensus in permissionless environments.  This is a technology that will power the SAFE Network, the only truly next-generation decentralised autonomous internet that promises privacy, security and freedom by default.
```
Comment (nadiaburborough) [inline src/contents/en-gb/parsec.yaml]: environments. This is ~a~ technology that will power the SAFE Network ~,~ `:`

File: src/contents/en-gb/parsec.yaml [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+networkUse:
+  title: Why does the Network need it?
+  para1: As a global decentralised data and communications network, the SAFE Network has to be public and permissionless.
+  para2: Unlike centralised systems there is no central body to act as a final authority on the state of the Network.  Nodes is a decentralised system such as the SAFE Network must therefore agree on Network actions and state amongst themselves. Other decentralised systems such as BitCoin use blockchain to achieve consensus.  However, there is a number of drawbacks with a blockchain based system notably the speed and cost as blockchain grows - this is not a scalable option.
+bigDeal:
+  title: Why is it a big deal?
+  para1: As far as we know no other project has managed to ensure ABFT consensus in permissionless environments.  This is a technology that will power the SAFE Network, the only truly next-generation decentralised autonomous internet that promises privacy, security and freedom by default.
+myProject:
+  title: Great, can I use it on my project?
```
Comment (nadiaburborough) [inline src/contents/en-gb/parsec.yaml]: can I use it ~on~ `in` my project?

File: src/contents/en-gb/parsec.yaml [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    video3:
+      name: "3. PARSEC - it's cool tech, yo! | SAFE Network - London Meetup"
+      href: 'https://www.youtube.com/watch?v=4p653ewa7R0'
+    video4:
+      name: "4. PARSEC Graphs - Visualising ABFT Consensus"
+      href: 'https://www.youtube.com/watch?v=kkafYeHKzIk&t=1s'
+resources:
+  title:
+    dark: PARSEC
+    light1: PARSEC Resources
```
Comment (nadiaburborough) [inline src/contents/en-gb/parsec.yaml]: different format to the other titles

File: src/constants.js [hunk 9]
```diff
@@ -20,6 +20,14 @@ export default {
         jams: 'https://safenetforum.org/t/introducing-jams-demo-alpha-2/21373',
         safeCms: 'https://safenetforum.org/t/safe-cms-censorship-is-a-thing-of-the-past/'
       }
+    },
+    parsec: {
+      resources: {
+        github: 'https://github.com/maidsafe/parsec',
+        rfc: 'https://github.com/maidsafe/rfcs/blob/master/text/0049-parsec/0049-parsec.md',
+        whitepaper: 'http://docs.maidsafe.net/Whitepapers/pdf/PARSEC.pdf',
```
Comment (S-Coyle) [inline src/constants.js]: Whitepaper URL should be updated to https

File: src/contents/en-gb/parsec.yaml [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+    light2: Resources For Developers
+  para: Resources for you to include PARSEC in your own projects.
+  res:
+    - name: GitHub
+      logo: GITHUB
+      linkId: github
+    - name: RFC
+      logo: RFC
```
Comment (S-Coyle) [inline src/contents/en-gb/parsec.yaml]: Tested the new /parsec page on osx Safari and noticed that the RFC, Whitepaper & Documentation images at the bottom of the page all flicker a lot when I scroll. The GitHub image is fine.
Not able to reproduce this on Chrome or FF, but confirmed a colleague could also repro on Safari

File: src/contents/en-gb/home.yaml (hunk 2 above)
Comment (S-Coyle) [reply src/contents/en-gb/home.yaml]: Reproduced this cross browser - it's also easy to resize and end up with the description overhanging the edge of the screen, when every other section of the page has resized appropriately. Or open the page in iPad Pro portrait dimensions - 1024x1366.

File: src/contents/en-gb/core_technology.yaml [hunk 11]
```diff
@@ -34,6 +34,12 @@ components:
         name: eventual consistency
         href: https://en.wikipedia.org/wiki/Eventual_consistency
       chunk3: "."
+    para3:
+      chunk1: In order to come to this eventual consistency the routing layer uses the consensus mechanism
+      link1:
+        name: PARSEC
+        href: '/parsec'
```
Comment (S-Coyle) [inline src/contents/en-gb/core_technology.yaml]: This opens the /parsec page in a new browser tab - just confirming whether this is intentional?

File: src/contents/en-gb/home.yaml (hunk 1 above)
Comment (david-maidsafe) [reply src/contents/en-gb/home.yaml]: adjusted (was the same diff as section above but looked bigger as section is smaller)

File: src/contents/en-gb/parsec.yaml (hunk 3 above)
Comment (david-maidsafe) [reply src/contents/en-gb/parsec.yaml]: it's the page header - I'll address (btw - just notes for me as I'm going through these)

File: src/contents/en-gb/parsec.yaml (hunk 4 above)
Comment (david-maidsafe) [reply src/contents/en-gb/parsec.yaml]: all but leading space done

File: src/contents/en-gb/parsec.yaml (hunk 5 above)
Comment (david-maidsafe) [reply src/contents/en-gb/parsec.yaml]: done

File: src/contents/en-gb/parsec.yaml (hunk 6 above)
Comment (david-maidsafe) [reply src/contents/en-gb/parsec.yaml]: done

File: src/contents/en-gb/parsec.yaml (hunk 7 above)
Comment (david-maidsafe) [reply src/contents/en-gb/parsec.yaml]: done

File: src/contents/en-gb/parsec.yaml (hunk 4 above)
Comment (david-maidsafe) [reply src/contents/en-gb/parsec.yaml]: format changed to match

File: src/contents/en-gb/parsec.yaml (hunk 3 above)
Comment (david-maidsafe) [reply src/contents/en-gb/parsec.yaml]: format changed to match

File: src/contents/en-gb/core_technology.yaml (hunk 11 above)
Comment (david-maidsafe) [reply src/contents/en-gb/core_technology.yaml]: wasn't just copied and pasted the link line from above but it doesn't make sense to open a new tab. Changed.

**Comment by david-maidsafe** [comment]
@S-Coyle & @nadiaburborough - I have addressed the points in your change requests. If you could confirm the changes and (on acceptance) would Stephen be able to do the merge?

File: src/contents/en-gb/home.yaml [hunk 12]
```diff
@@ -46,6 +46,18 @@ contribute:
...
       href: /core_technology
+sharing:
+  title:
+    shared: Stand alone
+    tech: technology
+    safe: SAFE Network
+    desc: Use standalone components from the SAFE Network in your own projects.
+  parsec:
+    title: PARSEC
+    desc: PARSEC (Protocol for Asynchronous, Reliable, Secure and Efficient Consensus) is an algorithm which allows a section of network nodes to reach consensus on the validity and order of network events by voting on events and using a gossip protocol to disseminate these votes amongst themselves.
```
Comment (nadiaburborough) [inline src/contents/en-gb/home.yaml]: Sorry to be picky .. but this sentence needs some commas and/or full stops to help the reader digest the info. Or provide just a summary sentence here (e.g. PARSEC (Protocol for Asynchronous, Reliable, Secure and Efficient Consensus) is an algorithm which allows nodes to reach consensus in a decentalised network.), and provide the fuller explanation on the page.

File: src/contents/en-gb/parsec.yaml [hunk 13]
```diff
@@ -0,0 +1,63 @@
+header:
+  title: PARSEC
+pageIntro:
+  title: What is PARSEC?
+  desc: PARSEC is a Protocol for Asynchronous, Reliable, Secure and Efficient Consensus and is the world's first open source ABFT protocol that can be used in a truly permissionless setting created by MaidSafe.
```
Comment (nadiaburborough) [inline src/contents/en-gb/parsec.yaml]: Suggest link from `ABFT protocol` to what that means

File: src/contents/en-gb/home.yaml (hunk 12 above)
Comment (david-maidsafe) [reply src/contents/en-gb/home.yaml]: Na, it's fair comment.

Comment (david-maidsafe) [reply src/contents/en-gb/home.yaml]: done

File: src/contents/en-gb/parsec.yaml (hunk 13 above)
Comment (david-maidsafe) [reply src/contents/en-gb/parsec.yaml]: Expanded out the initialism

**Comment by lionel1704** [comment]
@david-maidsafe from a technical perspective, the changes made are good to go!

File: src/constants.js (hunk 9 above)
Comment (S-Coyle) [reply src/constants.js]: Confirmed resolved

File: src/contents/en-gb/parsec.yaml (hunk 10 above)
Comment (S-Coyle) [reply src/contents/en-gb/parsec.yaml]: images have been removed from the page now - problem no longer visible.

**Comment by david-maidsafe** [comment]
@S-Coyle - made changes as requested and commit submitted.
Ready for Merge I guess - @shankar2105?

**Pull request merged**